src/tds/unittests/convert.c

```diff
@@ -38,6 +38,7 @@ free_convert(int type, CONV_RESULT *cr)
 {
 	switch (type) {
 	case SYBCHAR: case SYBVARCHAR: case SYBTEXT: case XSYBCHAR: case XSYBVARCHAR:
+	case SYBNVARCHAR: case SYBNTEXT: case XSYBNCHAR: case XSYBNVARCHAR:
```

Comment:
I honestly don't think tds_convert supports any of these. That's pretty weird. Also valgrind should have catch it. Do you have a reproduction?

Reply:
In retrospect, this is a fixup for 07a8138 from my other PR, even if it formally applies on its own.